Code/Editor/EditorFramework/Assets/Implementation/AssetDocument.cpp

```diff
@@ -370,19 +370,17 @@ ezStatus ezAssetDocument::TransformAsset(bool bTriggeredManually, const ezPlatfo
 
     const auto flags = GetAssetFlags();
     {
-      if (flags.IsAnySet(ezAssetDocumentFlags::DisableTransform | ezAssetDocumentFlags::OnlyTransformManually))
+      if (flags.IsSet(ezAssetDocumentFlags::DisableTransform) ||
+          (flags.IsSet(ezAssetDocumentFlags::OnlyTransformManually) && !transformFlags.IsSet(ezTransformFlags::TriggeredManually)))
         return ezStatus(EZ_SUCCESS, "Transform is disabled for this asset");
     }
   }
 
-  const ezStatus res = DoTransformAsset(pAssetProfile, bTriggeredManually);
+  const ezStatus res = DoTransformAsset(pAssetProfile, transformFlags);
 
   // some assets modify the document during transformation
   // make sure the state is saved, at least when the user actively executed the action
-  if (bTriggeredManually)
```

Comment:
Not a good idea to remove this. It would mean that back-ground processing (and also TransformAll) would suddenly save the document that you are working on ...

Reply:
Correct, put it back in.